src/libs/elektra/keyset.c

```diff
@@ -709,6 +709,38 @@ ssize_t ksSearchInternal (const KeySet * ks, const Key * toAppend)
 	}
 }
 
+/**
+ * Efficient search in a key set, either yielding the actual index
```

Comment:
Better to not promise something we don't do:

```suggestion
 *  Search key in a key set, either yielding the actual index
```

Reply:
I thought it would be the intended way this function works after the hash map lookup is being implemented. Removed `Efficient`